crates/bpf-filtering/src/initializer.rs

```diff
@@ -157,9 +184,37 @@ impl Initializer {
         if decision.interesting {
             log::debug!("tracking {} {}", process.pid, process.image);
         }
```

Comment:
I think this log can be moved inside `set_policy`

Reply:
`set_policy` doesn't have `process.image` because it is called also by `track_target_cgroups`, which doesn't have it. Not sure it's worth to look for the pid image just for a debug log.